idp/usso/usso.go

```
@@ -342,10 +342,13 @@ func (idp *identityProvider) getLaunchpadGroupsNoCache(ussoID string) ([]string,
 		return nil, errgo.Notef(err, "cannot get team list for launchpad user %q", user.Name())
 	}
 	groups := make([]string, 0, teams.TotalSize())
-	teams.For(func(team *lpad.Value) error {
+	err = teams.For(func(team *lpad.Value) error {
```

Fixes a bug where this error was not handled.